pandas/tests/reshape/test_pivot.py

```diff
@@ -116,7 +116,9 @@ def test_pivot_table_dropna_categoricals(self):
 
         result_false = df.pivot_table(index='B', columns='A', values='C',
                                       dropna=False)
-        expected_columns = Series(['a', 'b', 'c', 'd'], name='A')
+        expected_columns = (
+            Series(['a', 'b', 'c', 'd'], name='A').astype('category')
+        )
```

Comment:
how is this change related?

Reply:
Not sure :) The fact that it's only not a CategoricalIndex when in the columns and dropna=True seems like a bug on master.

```python
# This is on master
In [7]: df.pivot_table(index="B", columns="A", values="C", dropna=False).columns
Out[7]: Index(['a', 'b', 'c', 'd'], dtype='object', name='A')

In [8]: df.pivot_table(index="A", columns="B", values="C", dropna=False).index
Out[8]: CategoricalIndex(['a', 'b', 'c', 'd'], categories=['a', 'b', 'c', 'd'], ordered=False, name='A', dtype='category')
```

I've added a whatsnew.